Dockerfile

```diff
@@ -4,16 +4,12 @@ FROM golang:1.11.0-stretch
 RUN apt-get update && apt-get install -y make git unzip
 
 # should stay consistent with the version we use in Circle builds
-ARG PROTOC_VERSION=3.5.0
-# make a directory to do these operations in
-RUN export PROTOC_TMP_DIR=protoc && mkdir -p $PROTOC_TMP_DIR && cd $PROTOC_TMP_DIR \
-  # download the pre-built protoc binary
-  && curl --silent --show-error --location --output protoc.zip \
-    https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip \
-  # move the binary to /bin. move the well-known types ot /usr/local/include
-  && unzip protoc.zip && mv bin/protoc /bin/protoc && mv include/* /usr/local/include \
-  # remove all of the installation files
-  && cd .. && rm -rf $PROTOC_TMP_DIR
+ARG PROTOC_VERSION=3.6.1
+# download and install protoc binary and .proto files
+RUN curl --silent --show-error --location --output protoc.zip \
+  https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip \
+  && unzip -d /usr/local protoc.zip include/\* bin\/* \
```

Comment:
Minor nitpick: I'd restrict that to `bin/protoc`, but feel free to ignore :)

Reply:
IMO, it is actually better to use /usr/local folder same way than it is on CircleCI config.

Reply:
> I'd restrict that to `bin/protoc`

I can imagine some future version of `protoc` which calls another binary bundled in the same zip, let's say `protoc-helper`, and would not work without it, leaving a cryptic message, let's say `execve: no such file or directory`. With that in mind, I'd rather leave `bin/\*` in there.

Oh and I just spotted a bug here: it was `bin\/*` but should be `bin/\*`. Fixed.

Reply:
👍